### DISCUSSION TITLE
Why es7-persistence be removed?

### DISCUSSION THREAD

**Discussion by robert-guangzhou** [Netflix/conductor#3342] [Q&A]
I have saw es7 support at the doc:https://betterprogramming.pub/running-netflix-conductor-3-in-docker-using-elasticsearch-7-and-postgresql-b415988dd74a,  but I find   es7-persistence be remove from 3.8.x, why?

**Comment by v1r3n** [comment]
es7-persistence is available inside the community repo at https://github.com/Netflix/conductor-community
